src/code/utils/get-query-param.coffee

```diff
@@ -0,0 +1,9 @@
+module.exports = (param) ->
+  param = param.replace(/[\[]/, "\\[").replace(/[\]]/, "\\]")
+  regexS = "[\\?&]" + param + "=([^&#]*)"
+  regex = new RegExp regexS
+  results = regex.exec window.location.href
+  if results?.length > 1
+    return decodeURIComponent results[1]
+  else
+    return null
```

Comment:
Couldn't this be simplified to `results?.length > 1 ? decodeURIComponent results[1] : null`?